cli/bench/connect.go

```diff
@@ -0,0 +1,13 @@
+package bench
+
+import (
+	"github.com/FZambia/tarantool"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+func TarantoolConnect(ctx context.BenchCtx) (*tarantool.Connection, error) {
```

Comment:
It would be more logical if this function accepted the ``user string`` and the ``password string`` as input, and not the ``context.BenchCtx``.

Reply:
This time we can't resolve this situatuon with significant improvement in readability. In the future connection parameters may be added